demo/scripts/kind-cluster-config.yaml

```
@@ -14,8 +12,8 @@ nodes:
   - |
     kind: ClusterConfiguration
     apiServer:
-        extraArgs:
-          runtime-config: "resource.k8s.io/v1beta1=true"
+      extraArgs:
+        runtime-config: "resource.k8s.io/v1beta1=true"
```

If you expect this to work with kind clusters >= 1.32.0, then you probably want
```suggestion
      extraArgs:
        runtime-config: "resource.k8s.io/all=true,"
```